java/src/test/java/com/spotify/voyager/jni/StringIndexTest.java

```diff
@@ -239,6 +240,27 @@ public void itLoadsFromInputStreamNoParams() throws Exception {
     }
   }
 
+  @Test
+  public void indexResize() throws Exception {
```

Comment:
One small nit: Can we rename this test `itResizesIndex` for consistency with the other test names?

Reply:
done